src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter((movie) => {
+    if (
+      movie.title.toLowerCase().includes(query.toString()
+        .toLowerCase()) || movie.description.toLowerCase()
+        .includes(query.toString().toLowerCase())
+    ) {
+      return movie;
+    }
+
+    return false;
```

Comment:
a callback function being passed to `filter` method should return a boolean, what you return is `Movie | boolean`
let's make it more explicit and simpler, you don't even need to use if-statement